packages/lesswrong/lib/sql/SelectQuery.ts

```diff
@@ -148,6 +148,21 @@ class SelectQuery<T extends DbObject> extends Query<T> {
     }
   }
 
+  private selectorIsEmpty(selector?: string | MongoSelector<T>): boolean {
+    if (!selector) {
+      return true;
+    }
+
+    if (typeof selector === "string") {
+      return false;
+    }
```

Comment:
I assume that not checking for length here is an intentional change in behavior?

Reply:
If selector is "" the we returned true at lines 152/153